R/geom-sf.R

```diff
@@ -282,3 +278,11 @@ geom_sf_text <- function(mapping = aes(), data = NULL,
     layer_class = LayerSf
   )
 }
+
+sf_types <- c(GEOMETRY = "other", POINT = "point", LINESTRING = "line",
```

Comment:
IIUC, the types of simple feature supported by sf can be different in future. Can we just list up geometries of `point` and `line`, and treat others as `others`? (Not sure it works for unknown types, though)

Reply:
This was made in order to get a quick look-up. Most of these are not really supported by sf but just part of the simple feature spec. I'm quite sure sf won't begin to support something other than that in the next 10 years

Reply:
OK, it's a good reason. Let's talk about this again in 2029 :)